### DISCUSSION TITLE
[Xlsx-Import] What is the "index" variable?

### DISCUSSION THREAD

**Discussion by fnick851** [Siemienik/XToolset#164] [Q&A] [answered]
Index: 5 for author, index: 2 for title - what do these do?

**Comment by Siemienik** [comment, marked as answer]
For `type:'list'`the index indicates which column is requested.  

 Index | Column
--------|-------
 1 | `A`
 2 | `B`
 3 | `C`
...|...

@fnick851, I hope this enough explain what it is, if not please write me a message :smile:

btw. I decided to name it `index` instead of `co` because I am going to add supporting new `type: 'list-horizontal'` (https://github.com/Siemienik/xlsx-import/issues/7) which will work similar to `type:'list'` but it will read col by col instead of row by row. Then `index` will be translated to number of row.

**Comment by fnick851** [comment]
@Siemienik Thanks for the answer!
